manifests/init.pp

```diff
@@ -170,7 +155,7 @@
   Boolean $manage_repo                           = true,
   Boolean $manage_service                        = true,
   Boolean $restart_on_change                     = true,
-  Variant[Enum['systemd', 'launchd'], Boolean] $init_style = $facts['service_provider'],
+  Variant[Enum['systemd'], Boolean] $init_style  = $facts['service_provider'],
```

Comment:
I'd consider dropping this parameter altogether.

Reply:
i considered dropping it, but left it in so that folks could overwrite it at will. But i am happy to drop it